NewForce-Cohort-4/tabloid-full-stack-team-rose
# #9 View All Categories

Description

As an admin I would like to see all the available categories so that I can choose to edit or delete one, or see that I should add a new one.

Given an admin is in the app
When they select the Category Management link in the menu
Then they should be directed to a page that lists all the Category names ordered alphabetically

Type of change

 New feature (non-breaking change which adds functionality)

Testing Instructions
pull down af-viewcats. make sure api is running, run the react app. login with verified user. Click on Categories tab. You should see a list of all the available Categories. 

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
